WORKSPACE

```diff
@@ -384,11 +384,12 @@ http_archive(
     ],
 )
 
-http_file(
+http_archive(
     name = "libguestfs-appliance",
-    sha256 = "59fe17973fdaf4d969203b66b1446d855d406aea0736d06ee1cd624100942c8f",
+    sha256 = "0eaa28e0079ae9ac3bf495311ae437c6d55891ab2293de49b8c0061cfa0a6103",
     urls = [
-        "https://storage.googleapis.com/kubevirt-prow/devel/release/kubevirt/libguestfs-appliance/appliance-1.48.4-linux-5.14.0-176-centos9.tar.xz",
+        ## TODO: remove and replace when we release the appliance in qcow2 format
+        "file:///root/go/src/kubevirt.io/kubevirt/output/libguestfs-appliance-1.48.4-qcow2-linux-5.14.0-183-centos9.tar.xz",
```

Comment:
Proper appliance now available :)

```diff
diff --git a/WORKSPACE b/WORKSPACE
index 84c791bef..83f44187c 100644
--- a/WORKSPACE
+++ b/WORKSPACE
@@ -388,10 +388,9 @@ http_archive(
 
 http_archive(
     name = "libguestfs-appliance",
-    sha256 = "0eaa28e0079ae9ac3bf495311ae437c6d55891ab2293de49b8c0061cfa0a6103",
+    sha256 = "124d6325a799e958843be4818ef2c32661755be1c56e519665779948861b04f6",
     urls = [
-        ## TODO: remove and replace when we release the appliance in qcow2 format
-        "file:///root/go/src/kubevirt.io/kubevirt/output/libguestfs-appliance-1.48.4-qcow2-linux-5.14.0-183-centos9.tar.xz",
+        "https://storage.googleapis.com/kubevirt-prow/devel/release/kubevirt/libguestfs-appliance/libguestfs-appliance-1.48.4-qcow2-linux-5.14.0-183-centos9.tar.xz",
     ],
 )
 
```

With this, I think the PR can be marked as ready for review.